using-webhooks/client/ios-swift/app/CheckoutViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  CheckoutViewController.swift
+//  app
+//
+//  Created by Yuki Tokuhiro on 9/25/19.
+//  Copyright © 2019 stripe-samples. All rights reserved.
+//
+
+import UIKit
+import Stripe
+
+/**
+ * To run this app, you'll need to first run the sample server locally.
+ * Follow the "How to run locally" instructions in the root directory's README.md to get started.
+ * Once you've started the server, open http://localhost:4242 in your browser to check that the
+ * server is running locally.
+ * After verifying the sample server is running locally, build and run the app using the iOS simulator.
+ */
+let BackendUrl = "http://127.0.0.1:4242/"
+
+class CheckoutViewController: UIViewController {
+    var paymentIntentClientSecret: String?
+
+    lazy var cardTextField: STPPaymentCardTextField = {
+        let cardTextField = STPPaymentCardTextField()
+        return cardTextField
+    }()
+    lazy var payButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.layer.cornerRadius = 5
+        button.backgroundColor = .systemBlue
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 22)
+        button.setTitle("Pay", for: .normal)
+        button.addTarget(self, action: #selector(pay), for: .touchUpInside)
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        let stackView = UIStackView(arrangedSubviews: [cardTextField, payButton])
+        stackView.axis = .vertical
+        stackView.spacing = 20
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(stackView)
+        NSLayoutConstraint.activate([
+            stackView.leftAnchor.constraint(equalToSystemSpacingAfter: view.leftAnchor, multiplier: 2),
+            view.rightAnchor.constraint(equalToSystemSpacingAfter: stackView.rightAnchor, multiplier: 2),
+            stackView.topAnchor.constraint(equalToSystemSpacingBelow: view.topAnchor, multiplier: 2),
+        ])
+        loadPage()
+    }
+
+    func displayAlert(title: String, message: String, restartDemo: Bool = false) {
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+            if restartDemo {
+                alert.addAction(UIAlertAction(title: "Restart demo", style: .cancel) { _ in
+                    self.cardTextField.clear()
+                    self.loadPage()
+                })
+            }
+            else {
+                alert.addAction(UIAlertAction(title: "OK", style: .cancel))
+            }
+            self.present(alert, animated: true, completion: nil)
+        }
+    }
+
+    func loadPage() {
```

Comment:
nit: tbh this feels web-centric to me.  I'd rather call this `loadPaymentIntent` (or I think I called it `didStartCheckout` initially) 🤷‍♂ Could just be my personal taste though.

Reply:
I think that's fair. `didStartCheckout` felt a bit iOS-centric to me, but maybe `startCheckout` is a good middle ground!

Reply:
(I'm renaming this function to `startCheckout` in all the samples)